File: java-recipes/src/main/java/com/recipes/Ingredient.java
package com.recipes;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class Ingredient {
	public Integer ingredientId;
	public String name;
	public String quantity;
	public Integer calories;
}

File: fargopolis-web/src/models/Ingredient.ts
export default class Ingredient {
    ingredientId: number;
    name: string;
    quantity: string;
    calories: number;

    constructor(
        ingredientId: number,
        name: string,
        quantity: string,
        calories: number
    ) {
        this.ingredientId = ingredientId;
        this.name = name;
        this.quantity = quantity;
        this.calories = calories;
    }

}